src/core/entities/Trade.spec.ts

```diff
@@ -10,13 +10,13 @@ import { WeiAsToken } from './Wei'
 describe('Trade', () => {
   test('exact in', () => {
     const TokenA = new TokenImpl({
-      address: parseAddress('0xb9920BD871e39C6EF46169c32e7AC4C698688881'),
+      address: parseAddress('0xb9920bd871e39c6ef46169c32e7ac4c698688881'),
```

Comment:
Tests don't care about exact addresses, here could be just ANY valid address. For what do you change them here?

Reply:
Ok, I've left change in only 1 test that failed with uppercase